easybuild/tools/filetools.py

```diff
@@ -2270,6 +2270,8 @@ def copy_file(path, target_path, force_in_dry_run=False):
     :param target_path: path to copy the file to
     :param force_in_dry_run: force copying of file during dry run
     """
+    if not os.path.exists(path):
```

Comment:
To avoid breaking the use case of copying a broken symlink (see broken test), you should add a `not os.path.islink(path)` here?

Reply:
Fixed in latest commit